packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js

```diff
@@ -443,6 +443,44 @@ async function transformGraphQLSchema(context, options) {
 
   if (directiveMap.directives.includes('searchable')) {
     searchableTransformerFlag = true;
+    if (context.input.command === 'add') {
```

Comment:
We could also use`resourcesToBeCreated` here similar to apiProjectIsFromOldVersion.
https://github.com/aws-amplify/amplify-cli/blob/16a2b4f50a85372f733eef129e43b154a993b136/packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js#L219